storage/framework/onboarding/index.ts

```diff
@@ -0,0 +1,79 @@
+import { setTimeout } from 'node:timers/promises'
+import * as p from '@clack/prompts'
+import color from 'picocolors'
+import { runCommand } from '@stacksjs/cli'
+import { path } from '@stacksjs/path'
+
+async function main() {
+  console.clear()
+
+  await setTimeout(1000)
+
+  p.intro(`Stacks ${color.bgCyan(color.black(' create-app '))}`)
+
+  const defaultFolderPath = './stacks'
+
+  const project = await p.group(
+    {
+      path: () =>
+        p.text({
+          message: 'Where should we create your project?',
+          initialValue: defaultFolderPath,
+          placeholder: defaultFolderPath,
+          validate: (value) => {
+            if (!value)
+              return 'Please enter a path.'
+            if (value[0] !== '.')
+              return 'Please enter a relative path.'
+          },
+        }),
+      type: ({ results }) =>
+        p.select({
+          message: `Pick your flavor ("${results.path}")`,
+          initialValue: 'default',
+          maxItems: 3,
+          options: [
+            { value: 'default', label: 'Default (UI with API)' },
+            { value: 'ui', label: 'UI' },
+            { value: 'api', label: 'API' },
+          ],
+        }),
+      modules: () =>
+        p.multiselect({
+          message: 'Select additional modules.',
+          initialValues: ['prettier', 'eslint'],
```

Comment:
Unsure why prettier is here. We don't use it, and actively suggest people not to use it in combination with Stacks formatting 👍